src/types.ts

```diff
@@ -19,7 +21,10 @@ export interface ServerConfig {
 }
 
 export interface ClientConfig {
-  mcpServers: Record<string, ServerConfig>;
+  mcpServers?: Record<string, ServerConfig>;
+  mcp?: {
+    servers: Record<string, ServerConfig>;
+  };
```

Comment:
Imo we may want to 'normalize' this and keep `mcpServers` for everyone, just modify it when it's written into vscode's settings